examples/extensions/authorizer.middleware.ts

```diff
@@ -0,0 +1,43 @@
+import { GraphQLResolveInfo, GraphQLObjectType } from "graphql";
+
+import { MiddlewareFn } from "../../src";
+import { Context } from "./context.interface";
+import { UnauthorizedError } from "../../src/errors";
+
+const extractAuthorizationExtensions = (info: GraphQLResolveInfo) => {
```

Comment:
I would love to get some feedback on this, I am not so experienced with schema inspection and I am wondering if there is a more elegant way to do this? I was expecting it to be more easy to "read" the meta data from the field being resolved.

The order in which I am overriding the extensions is arbitrary of course.

I am genuinely interested if there is a better way since we will be using this feature for authorization ourselves 🙂

Reply:
I tried to do this in this commit: cf7aa50de2207adff88a28b0f54b4411b34b3647
it seems a bit more complicated to me as we are not really interested in the whole config here and just the extensions though 🤔

Reply:
Yes, but for other extensions usage we might reuse that helpers, that's why it's better to have them extracted. It's also easier to read than the 20 lines monolith.